packages/cojson/src/coValueCore.ts

```diff
@@ -126,10 +126,7 @@ export class CoValueCore {
         .expectCoValueLoaded(header.ruleset.group)
         .subscribe((_groupUpdate) => {
           this._cachedContent = undefined;
-          const newContent = this.getCurrentContent();
-          for (const listener of this.listeners) {
-            listener(newContent);
-          }
+          this.notifyUpdate("immediate");
         });
```

Comment:
We have already a test on this logic to ensure that I'm not intorducing a regression.

Checking if we can optimize this block by resetting the cache only when required (updates on the readKey and getting the read access)